wire-ios-sync-engine/Tests/Source/Synchronization/Transcoders/ZMMissingUpdateEventsTranscoderTests.m

```diff
@@ -52,7 +53,11 @@ - (void)setUp {
     
     self.requestSync = [OCMockObject mockForClass:ZMSingleRequestSync.class];
     self.syncStateDelegate = [OCMockObject niceMockForProtocol:@protocol(ZMSyncStateDelegate)];
-    self.mockSyncStatus = [[MockSyncStatus alloc] initWithManagedObjectContext:self.syncMOC syncStateDelegate:self.syncStateDelegate];
+    self.lastEventIDRepository = [[LastEventIDRepository alloc] initWithUserID:self.userIdentifier
+                                                                                    userDefaults:NSUserDefaults.standardUserDefaults];
```

Comment:
same

Reply:
fixed in c8a59a522c5b2a06d5a9bf871a2a7e174175247d